tests/mir_test_framework/executable_path.cpp

```diff
@@ -44,16 +44,16 @@ std::string mir_test_framework::executable_path()
 
 std::string mir_test_framework::library_path()
 {
-    static char libpath[1024];
+    static std::string libpath;
 
-    if (!libpath[0])
+    if (libpath.empty())
```

Comment:
Is this not always true?

Reply:
Not following the assignment that takes place on line 56